### ISSUE TITLE
Quick question about VGG19

### ISSUE THREAD

**Issue by zzhan127** [deepak112/Keras-SRGAN#10]
Hello, 

thank you for your work on SRGAN. I am working on using SRGAN for medical images. 
I have a question about VGG19.  The VGG19 is trained on a large dataset of color images, so if I want to use SRGAN for grayscale medical images, can we still use VGG19 for the loss?

Thank you!

**Comment by deepak112** [comment]
Hi,

Yes we can use it for medical datasets with grayscale images. I also used
it for CCTV grayscale images.

Regards
Deepak

**Comment by zzhan127** [comment]
Great. Thanks for the help.
